assets/src/console/pages/payments/RefundPage.tsx

```diff
@@ -105,77 +108,82 @@ const RefundPage: React.FC<Props> = ({
           showError
```

Comment:
@acodegirl I just noticed that there is copy/paste error here from a previous refactor `Unable to retrieve events` that should probably be `Unable to submit refund`—might be worth fixing that up while you're in here.